battle.net/README.md

```diff
@@ -0,0 +1,29 @@
+# Battle.net chocolatey package
+
+### Pre-Requisites
+
+This package requires the `autohotkey` chocolatey package
+This package requires the `chocolatey-misc-helpers.extension` chocolatey package
+
+The updater requires a properly configured installation of [AU](https://github.com/majkinetor/au)

```

Comment:
Probably best to mention [chocolatey-au](https://github.com/chocolatey-community/chocolatey-au) here as well.